lib/Saml2/Utils.php

```diff
@@ -348,6 +389,26 @@ public static function setSelfHost($host)
     }
 
     /**
+     * @param $baseurlpath string The baseurl path to use when constructing URLs
+     */
+    public static function setBaseURLPath($baseurlpath)
+    {
+        if (!isset($baseurlpath) || empty($baseurlpath)) {
```

Comment:
you can just used `empty` as `isset` will only return false if `$baseurlpath` is explicitly `null`, but `empty` will return true for `null` anyway.